MAUI/PDF-Viewer/Show-Hide.md

```diff
@@ -0,0 +1,83 @@
+
+## Hide annotations
+
+To hide annotations in the document, set the `Hidden` property of the annotations to `true`.
+
+### Hide all annotations 
+
+While reviewing and presenting documents, all annotations can be hidden to improve the focus on the content and to avoid distraction caused by cluttered annotations. The following example explains how to hide all annotations in a PDF document. 
+
+{% tabs %}
+{% highlight xaml %}
+<Grid> 
+    <Grid.RowDefinitions> 
+        <RowDefinition Height="Auto" /> 
+        <RowDefinition Height="*" /> 
+    </Grid.RowDefinitions> 
+    <Button Text="HideAllAnnotations" Clicked="HideAllAnnotationButton_Clicked" VerticalOptions="Center" HorizontalOptions="Center" Grid.Row="0"/> 
+    <syncfusion:SfPdfViewer x:Name="PdfViewer" DocumentSource="{Binding PdfDocumentStream}" Grid.Row="1" /> 
+</Grid> 
+{% endhighlight %}
+
+{% highlight C# %}
+private void HideAllAnnotationsButton_Clicked(object sender, EventArgs e) 
+{ 
+    ReadOnlyObservableCollection<Annotation> annotations=PdfViewer.Annotations; 
+    foreach(Annotation annotation in annotations) 
+    { 
+        annotation.Hidden = true; 
+    } 
+} 
+{% endhighlight %}
+{% endtabs %}
+
+* Similarly, to show all the annotations, set the Hidden property value to `false`.
+
+### Hide specific annotation 
+
+While collaborating, the `Hidden` property of annotations can be used to show and hide annotations from specific authors. The following example illustrates how to hide annotations from a specific author by accessing the annotation's 'Hidden' property. 
+
+{% tabs %}
+{% highlight xaml %}
+<Grid> 
+    <Grid.RowDefinitions> 
+        <RowDefinition Height="Auto" /> 
+        <RowDefinition Height="*" /> 
+    </Grid.RowDefinitions> 
+    <Button Text="HideAnnotation" Clicked="HideAnnotationButton_Clicked" VerticalOptions="Center" HorizontalOptions="Center" Grid.Row="0"/> 
+    <syncfusion:SfPdfViewer x:Name="PdfViewer" DocumentSource="{Binding PdfDocumentStream}" Grid.Row="1" /> 
+</Grid> 
+{% endhighlight %}
+
+{% highlight C# %}
+private void HideAnnotationButton_Clicked(object sender, EventArgs e) 
+{ 
+  ReadOnlyObservableCollection<Annotation> annotationList = PdfViewer.Annotations; 
+  foreach (Annotation annotation in annotationList) 
+  { 
+     if (annotation.Author == "John") 
+        { 
+            annotation.Hidden = true; 
+        } 
+  }            
+}
+{% endhighlight %}
+{% endtabs %}
+
+* Similarly, to show the annotation, set the `Hidden` property value to `false`. 
+
+## Note:
```

Comment:
Note should be added as tag, not as header. Please cross check existing md files